Tests/Builder/FormContractorTest.php

```diff
@@ -46,4 +47,68 @@ public function testGetFormBuilder()
             $this->formContractor->getFormBuilder('test', array('foo' => 'bar'))
         );
     }
+
+    public function testDefaultOptionsForSonataFormTypes()
+    {
+        $admin = $this->getMockBuilder('Sonata\AdminBundle\Admin\AdminInterface')->getMock();
```

Comment:
> I don't remember why we don't use phpunit 4.8 for every build.

PHPUnit 4.8 is not **fully** compatible with PHP7. You can have some not fixed edge case I think.